lib/cfnguardian/resources/application_targetgroup.rb

```diff
@@ -23,9 +23,12 @@ def default_alarms
       alarm = CfnGuardian::Models::ApplicationTargetGroupAlarm.new(@resource)
       alarm.name = 'TargetResponseTime'
       alarm.metric_name = 'TargetResponseTime'
-      alarm.threshold = 5
+      alarm.threshold = 30
       alarm.evaluation_periods = 5
-      alarm.treat_missing_data = 'notBreaching'
+      alarm.period = 60
```

Comment:
60 is the default value set in the base class
https://github.com/base2Services/cfn-guardian/blob/master/lib/cfnguardian/models/alarm.rb#L43